pkgs/applications/kde/default.nix

```diff
@@ -88,6 +88,7 @@ let
       gwenview = callPackage ./gwenview.nix {};
       incidenceeditor = callPackage ./incidenceeditor.nix {};
       k3b = callPackage ./k3b.nix {};
+      kaccounts-integration = callPackage ./kaccounts-integration.nix {};
```

Comment:
This must also be added to the big `inherit` block in `mkLibsForQt5`.

Reply:
Addressed in 0bc5689058dde8e219ce81356e7f890245529858